api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -445,6 +448,45 @@ private void verifyInvitationKey(String invitationKey){
     }
   }
 
+  @Override
+  public ResponseEntity<Void> updateContactEmail(UpdateContactEmailRequest updateContactEmailRequest) {
+    verifyNoLogins(updateContactEmailRequest.getUsername());
+    //resend Welcome Email to new contact email... Add once Mandrill API complete
```

Comment:
was planning on using the resendWelcomeEmail method from below here (once I had that working the way I wanted)